builder/ibmcloud/vpc/step_get_subnet_info.go

```diff
@@ -34,6 +34,14 @@ func (s *stepGetSubnetInfo) Run(_ context.Context, state multistep.StateBag) mul
 	}
 
 	vpcId := *subnetData.VPC.ID
+	secGrpVPC := state.Get("user_sec_grp_vpc")
```

Comment:
This is throwing an error saying security group is not in same vpc. Even when not passing sg id